spec/lib/miq_expression_spec.rb

```diff
@@ -9,14 +9,18 @@
          display_range
          entity
          tag_name
+         tenant_name
          label_name
          id
          vm_id
          vm_name)
     end
 
     it 'lists custom attributes in ChargebackVm' do
-      skip('removing of virtual custom attributes is needed to do first in other specs')
+      # sometimes this is called before this, other times not
+      # this can be invoked by running a chargeback report like we in have specs:
+      # https://github.com/ManageIQ/manageiq/blob/1a4de9c0193651c3bc6d2e6c44fdc1429a61c6fe/spec/models/miq_report_spec.rb#L1131
+      ChargebackVm.load_custom_attributes_for(['virtual_custom_attribute_attr_1'])
```

Comment:
it can work and in addition method `reporting_available_fields` uses method [<class>.custom_keys](https://github.com/ManageIQ/manageiq/blob/14e100cf68085524e229ccdf34fe8acdfa82d85d/app/models/mixins/custom_attribute_mixin.rb#L32) to list virtual attributes and as you can see it requires existence `CustomAttribute` record.

so It looks like that we need to also create custom attribute record , with `name = attr_1' here and together with this change - it could be reason CI is failing.